examples/website/icon/app.tsx

```diff
@@ -24,7 +24,10 @@ const INITIAL_VIEW_STATE = {
 
 const MAP_STYLE = 'https://basemaps.cartocdn.com/gl/dark-matter-nolabels-gl-style/style.json';
 
-function renderTooltip(info) {
+function renderTooltip(info: PickingInfo & {objects: any} | null) {
+  if (!info?.object) {
+    return null;
+  }
```

Comment:
This change isn't obvious to me — seems like `info.object` would never be used unless we get past the `info.objects` check below. Do you mind adding an inline comment about the reason for this early exit?

Reply:
You're right, I've move the check back down to where it was